acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuNodeConfigBuilder.java

```diff
@@ -317,6 +317,18 @@ public TekuNodeConfigBuilder withWritableKeystorePath(
     return this;
   }
 
+  public TekuNodeConfigBuilder withDefaultLogging() {
+    LOG.debug("Removed log-destination from config");
+    configMap.remove("log-destination");
+    return this;
+  }
+
+  public TekuNodeConfigBuilder withValidatorKeysPath(final String path) {
```

Comment:
all of the other functions setting validator keys are named differently, and i think this needs to be consistent so people don't find this one and expect it to work...
We currently have
`withReadOnlyKeystorePath`
`withWritableKeystorePath`

I'd rename this function to 
`withInvalidKeystorePath`

If someone uses this function instead of the others, the files won't be present etc.

I'd also copy the debug mostly so 
`LOG.debug("set validator-keys folder to INVALID {}", path);`